blueprints/data-solutions/data-platform-minimal/variables.tf

```diff
@@ -0,0 +1,225 @@
+        memory_gb  = 1.875
+        storage_gb = 1
+        count      = 1
+      }
+      web_server = {
+        cpu        = 0.5
+        memory_gb  = 1.875
+        storage_gb = 1
+      }
+      worker = {
+        cpu        = 0.5
+        memory_gb  = 1.875
+        storage_gb = 1
+        min_count  = 1
+        max_count  = 3
+      }
+    }
+  }
+}
+
+variable "data_catalog_tags" {
+  description = "List of Data Catalog Policy tags to be created with optional IAM binging configuration in {tag => {ROLE => [MEMBERS]}} format."
+  type        = map(map(list(string)))
+  nullable    = false
+  default = {
+    "3_Confidential" = null
+    "2_Private"      = null
+    "1_Sensitive"    = null
+  }
+}
+
+variable "data_force_destroy" {
+  description = "Flag to set 'force_destroy' on data services like BiguQery or Cloud Storage."
+  type        = bool
+  default     = false
+}
+
+variable "groups" {
+  description = "User groups."
+  type        = map(string)
+  default = {
+    data-analysts  = "gcp-data-analysts"
+    data-engineers = "gcp-data-engineers"
+    data-security  = "gcp-data-security"
+  }
+}
+
+variable "location" {
+  description = "Location used for multi-regional resources."
+  type        = string
+  default     = "eu"
+}
+
+variable "network_config" {
+  description = "Shared VPC network configurations to use. If null networks will be created in projects with preconfigured values."
+  type = object({
+    host_project      = string
+    network_self_link = string
+    subnet_self_links = object({
+      processing_dataproc = string
+      processing_composer = string
+    })
+    composer_ip_ranges = object({
+      cloudsql   = string
+      gke_master = string
+    })
+    composer_secondary_ranges = object({
+      pods     = string
+      services = string
+    })
+    # web_server_network_access_control = list(string)
+  })
+  default = null
+}
+
+variable "organization_domain" {
+  description = "Organization domain."
+  type        = string
+}
+
+variable "prefix" {
+  description = "Prefix used for resource names."
+  type        = string
+  validation {
+    condition     = var.prefix != ""
+    error_message = "Prefix cannot be empty."
+  }
+}
+
+variable "project_config" {
+  description = "Provide 'billing_account_id' value if project creation is needed, uses existing 'project_ids' if null. Parent is in 'folders/nnn' or 'organizations/nnn' format."
+  type = object({
+    billing_account_id = optional(string, null)
+    parent             = string
+    project_ids = optional(object({
+      landing    = string
+      processing = string
+      curated    = string
+      common     = string
+      }), {
+      landing    = "lnd"
+      processing = "prc"
+      curated    = "cur"
+      common     = "cmn"
+      }
+    )
+  })
+  validation {
+    condition     = var.project_config.billing_account_id != null || var.project_config.project_ids != null
+    error_message = "At least one attribute should be set."
```

Comment:
`At least one of project_config.billing_account_id or var.project_config.project_ids should be set `